packages/prisma-client/migrations-cli/commands.ts

```diff
@@ -129,6 +129,26 @@ const ensureLastPending = async (migrationsName: string) => {
   }
 };
 
+const isNoopSql = (sqlScript: string) => {
```

Comment:
you probably wanna split these into a separate pr and we can merge it quickly

Reply:
Sorry for mixing this up. I'll do better next time, but I'd safe time on splitting and merge this as is